core/src/main/scala/sttp/tapir/Codec.scala

```diff
@@ -91,7 +93,50 @@ object Codec extends MultipartCodecDerivation with FormCodecDerivation {
 
   def stringCodec(charset: Charset): PlainCodec[String] = plainCodec(identity, charset)
 
-  private def plainCodec[T: Schema](parse: String => T, charset: Charset = StandardCharsets.UTF_8): PlainCodec[T] =
+  implicit val localTimePlainCodec: PlainCodec[LocalTime] = plainCodec[LocalTime](LocalTime.parse)
+  implicit val localDatePlainCodec: PlainCodec[LocalDate] = plainCodec[LocalDate](LocalDate.parse)
+
+  implicit val zonedDateTimePlainCodec: PlainCodec[ZonedDateTime] =
+    new PlainCodec[ZonedDateTime] {
```

Comment:
code organization idea: this might get long, so maybe we could categorize the codecs and e.g. put the datetime-related ones into a `trait DateTimeCodec`, which would be extended by `Codec`? Similarly for other categories

Reply:
I think it would require us to extract a `BaseCodec` with basically all that was in `Codec` so far, plus `DateTimeCodec extends BaseCodec` with new datetime stuff.

Reply:
hm right ... maybe not a good idea then ;)